docs/scos/dev/feature-integration-guides/202307.0/glue-api/data-exchange-api/data-exchange-api-integration.md

```diff
@@ -302,6 +396,16 @@ class DocumentationGeneratorApiDependencyProvider extends SprykerDocumentationGe
     {	    
         $contextExpanderCollection->addExpander(new DynamicEntityApiSchemaContextExpanderPlugin(), [static::GLUE_BACKEND_API_APPLICATION_NAME]);
     }
+    
+    /**
+     * @return array<\Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\Plugin\DocumentationInvalidationVoterPluginInterface>
+     */
+    protected function getInvalidationVoterPlugins(): array
```

Comment:
can we add a bit more context here what is this and how people can use it? E.g. what is this stack is for.